rest/data/project.go

```diff
@@ -259,6 +260,27 @@ func (pc *DBProjectConnector) UpdateProjectVars(projectId string, varsModel *res
 	return nil
 }
 
+func (pc *DBProjectConnector) UpdateProjectVarsByValue(toReplace string, replacement string) ([]*db.ChangeInfo, error) {
+	matchingProjects, err := model.GetVarsByValue(toReplace)
+	if matchingProjects == nil {
+		return nil, errors.Wrap(err, "no projects with matching value found")
```

Comment:
can't wrap error here, because you should check error separately, so this would be a new error instead